examples/window-app/common/window-app.zap

```diff
@@ -1142,22 +1292,48 @@
               "reportableChange": 0
             },
             {
-              "name": "ActiveCalendarType",
+              "name": "ClusterRevision",
+              "code": 65533,
+              "mfgCode": null,
+              "side": "client",
+              "included": 1,
+              "storageOption": "RAM",
+              "singleton": 0,
+              "bounded": 0,
+              "defaultValue": "1",
+              "reportable": 1,
+              "minInterval": 1,
+              "maxInterval": 65534,
+              "reportableChange": 0
+            }
+          ]
+        },
+        {
+          "name": "Localization Configuration",
+          "code": 43,
+          "mfgCode": null,
+          "define": "LOCALIZATION_CONFIGURATION_CLUSTER",
+          "side": "server",
+          "enabled": 1,
+          "commands": [],
+          "attributes": [
+            {
+              "name": "ActiveLocale",
               "code": 1,
               "mfgCode": null,
               "side": "server",
               "included": 1,
               "storageOption": "NVM",
               "singleton": 0,
               "bounded": 0,
-              "defaultValue": "0",
+              "defaultValue": "en-US",
               "reportable": 1,
               "minInterval": 1,
               "maxInterval": 65534,
               "reportableChange": 0
             },
             {
-              "name": "SupportedCalendarTypes",
+              "name": "SupportedLocales",
               "code": 2,
```

Comment:
@bzbarsky-apple how can we declare a list of supported locales in Zap directly ?

Reply:
Boris can correct me if I am wrong, but I believe you would do this by doing a configure on the "Localization Configuration" and then entering into the text field  in the attributes UI.

Reply:
ok this is unfortunate then we need to do tackle this is the code...